src/main/java/frc/robot/RobotContainer.java

```diff
@@ -267,9 +279,20 @@ public RobotContainer() {
     private void configureButtonBindings() {
         driveController.getBalancerButton()
             .onTrue(new InstantCommand(() -> {
-                balancerCommand = new BalancerCommand(driveSubsystem);
+                balancerCommand.reachedStation = false;
```

Comment:
Seems like this should go in `BalancerCommand.initialize()` instead of another callback on the button.